### PR TITLE
created entry points

### PR DESCRIPTION
**Pull request by seblapela**
* #429

### CODE REVIEW

File: source/databricks/setup.py [hunk 1]
```diff
@@ -31,7 +31,11 @@
         "console_scripts": [
             "start_stream = package.integration_events_persister_streaming:start",
             "start_calculator = package.calculator_job:start",
-            "begin_migration = package.datamigration.begin_migration:start",
+            "lock_storage = package.datamigration.lock_storage:lock",
+            "unlock_storage = package.datamigration.lock_storage:unlock",
+            "stop_db_jobs = package.datamigration.jobs:stop_db_jobs",
+            "start_db_jobs = package.datamigration.jobs:start_db_jobs",
+            "start_migrations = package.datamigration.migration:start_migrations",
```
Comment (BjarkeMeier) [inline source/databricks/setup.py:38]: How about this?

```suggestion
            "migrate_data_lake = package.datamigration.migration:start_migrations",
```

Comment (seblapela) [reply source/databricks/setup.py:38]: Thats ok

File: source/databricks/tests/entry_points/test_entry_points.py [hunk 2]
```diff
@@ -18,3 +18,64 @@
...
+        [
+            "start_db_jobs",
+            "--data-storage-account-name",
+            "foo",
+            "--data-storage-account-key",
+            "foo",
+            "--integration-events-path",
+            "foo",
+            "--process-results-path",
+            "foo",
```
Comment (BjarkeMeier) [inline source/databricks/tests/entry_points/test_entry_points.py:57]: Needed?

File: source/databricks/tests/entry_points/test_entry_points.py [hunk 3]
```diff
@@ -18,3 +18,64 @@
...
+        [
+            "stop_db_jobs",
+            "--data-storage-account-name",
+            "foo",
+            "--data-storage-account-key",
+            "foo",
+            "--integration-events-path",
+            "foo",
+            "--process-results-path",
+            "foo",
```
Comment (BjarkeMeier) [inline source/databricks/tests/entry_points/test_entry_points.py:34]: Needed?

**Pull request merged**